src/EnergyPlus/SizingManager.cc

```diff
@@ -805,6 +807,7 @@ void ManageSizing(EnergyPlusData &state)
                 coolPeakDDDate = SysSizPeakDDNum(AirLoopNum).cTotCoolPeakDDDate;
                 coolPeakDD = SysSizPeakDDNum(AirLoopNum).TotCoolPeakDD;
                 coolCap = FinalSysSizing(AirLoopNum).TotCoolCap;
+                timeStepIndexAtPeakCoolLoad = SysSizPeakDDNum(AirLoopNum).TimeStepAtTotCoolPk(coolPeakDD);
             }
```

Comment:
Defined a local variable `timeStepIndexAtPeakCoolLoad` that holds the index to the system cooling `peak time` depending on the `Type of Load to Size On` user input.